config/feature_flags.yml

```
@@ -1,4 +1,14 @@
 feature_flags:
+  enable_login_with_azure:
+    author: Richard Pattinson
+    description: Allow users to use the login with azure feature.
```

```suggestion
    description: Allow users to sign in using accounts in Active Directory.
```

The functionality is part of Azure, but it's specifically signing in with accounts stored in Active Directory.